R/roclet-doxygen.R

```diff
@@ -0,0 +1,138 @@
+#' @title Replace a value for a given tag on file in memory
+#' @description Scan the lines and change the value for the named tag if one line has this tag, 
+#'    add a line at the end if no line has this tag and return a warning if several lines
+#'    matching the tag
+#' @param fileStrings A vector with each string containing a line of the file
+#' @param tag The tag to be searched for 
+#' @param newVal The new value for the tag
+#' @return The vector of strings with the new value
+#' @examples
+#' fakeFileStrings <- c("Hello = world","SURE\t= indeed","Hello = you")
+#' 
+#' expect <- warning(ReplaceTag(fakeFileStrings,"Hello","me"))
+#' 
+#' newFake <- ReplaceTag(fakeFileStrings,"SURE","me")
+#' expect <- equal(length(newFake), length(fakeFileStrings))
+#' expect <- equal(length(grep("SURE",newFake)), 1)
+#' expect <- equal(length(grep("me",newFake)), 1)
+#' 
+#' newFake <- ReplaceTag(fakeFileStrings,"Bouh","frightened?")
+#' expect <- equal(length(newFake), length(fakeFileStrings)+1)
+#' expect <- equal(length(grep("Bouh",newFake)), 1)
+#' expect <- equal(length(grep("frightened?",newFake)), 1)
+#' @keywords internal 
```

Comment:
I moved the examples to tests and removed the ' otherwise so that it doesn't appear in the documentation while remaining as comments.